explorer/src/__tests__/realtime.test.ts

```diff
@@ -182,4 +189,28 @@ describe('realtime', () => {
       },
     )
   })
+
+  it('rejects multiple connections from single node', async done => {
+    expect.assertions(4)
+
+    const ws1: WebSocket = await newChainlinkNode(
+      ENDPOINT,
+      chainlinkNode.accessKey,
+      secret,
+    )
+    const ws2: WebSocket = await newChainlinkNode(
+      ENDPOINT,
+      chainlinkNode.accessKey,
+      secret,
+    )
+
+    ws1.addEventListener('close', (event: WebSocket.CloseEvent) => {
+      expect(ws1.readyState).toBe(CLOSED_CONNECTION)
```

Comment:
Looking good so far @RyanRHall.

I feel like L#208 & L#209 should using the existing constants i.e. `WebSocket.CLOSED, WebSocket.OPEN`

Reply:
Ahh great catch. Didn't realize those existed. Thanks @rupurt!

Reply:
Change made.